app_layout_1.R

```diff
@@ -7,19 +7,22 @@ library(dashCoreComponents)
 library(dashBootstrapComponents)
 library(ggplot2)
 library(plotly)
+source("./src/app/app_wrangling.R")
 
 
 # * WRANGLING - load board game data here
-
-
+boardgame_data <- call_boardgame_data()
 
 # * WRANGLING - get dictionary for dropdowns here
+col_key_list <- c("category", "mechanic", "publisher")
+col_dict <- vector(mode="list", length=3)
+names(col_dict) <- col_key_list
 
+for (idx in 1:3){
```

Comment:
sub optimal, couldn't figure out a map right away
ill keep working on it
and im open to suggestions ; )

Reply:
lets get it all working into the `dasjboard_layout` branch and then do clean-up before into `main`, for loop all good at this level